video_xblock/video_xblock.py

```diff
@@ -116,6 +116,12 @@ class VideoXBlock(StudioEditableXBlockMixin, XBlock):
         help=_("Video muted or not")
     )
 
+    transcript_enabled = Boolean(
```

Comment:
`transcripts_enabled` to be consistent with `transcripts` field.

Also there may be more that one transcript.